src/scss/components/button/_index.scss

```diff
@@ -1 +1,315 @@
+    justify-content: center;
+
+    > * + * {
+      margin-left: 0.5rem;
+    }
+
+    &:focus {
+      outline: none;
+    }
+
+    &[disabled] {
+      cursor: not-allowed;
+
+      &:focus {
+        box-shadow: none;
+      }
+    }
+  }
+
+  @mixin primary() {
+    background-color: $cyan-900;
+    color: $gray-0;
+
+    &:hover {
+      background-color: $cyan-500;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:active {
+      background-color: $cyan-700;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+      outline: none;
+    }
+
+    &[disabled] {
+      background-color: $gray-400;
+      border-color: transparent;
+      color: $gray-700;
+    }
+  }
+
+  @mixin secondary() {
+    border: 1px solid $cyan-900;
+    color: $cyan-900;
+
+    &:hover {
+      background-color: $cyan-500;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:active {
+      background-color: $cyan-700;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+    }
+
+    &[disabled] {
+      background-color: transparent;
+      border-color: $gray-400;
+      color: $gray-400;
+    }
+  }
+
+  @mixin tertiary() {
+    background-color: transparent;
+    border: none;
+    color: $cyan-900;
+    padding-left: 0;
+    padding-right: 0;
+
+    &:hover {
+      background-color: transparent;
+      border-color: transparent;
+      color: $cyan-500;
+    }
+
+    &:active {
+      background-color: transparent;
+      border-color: transparent;
+      color: $cyan-700;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+    }
+
+    &[disabled] {
+      background-color: transparent;
+      border-color: transparent;
+      color: $gray-400;
+    }
+  }
+
+  @mixin small() {
+    padding: 0.125rem 0.5rem; // 2px 8px
+  }
+
+  @mixin medium() {
+    font-size: 0.875rem;
+    padding: 8px 12px;
+  }
+
+  @mixin large() {
+    font-size: 1rem;
+    padding: 1rem;
+  }
+}
+
+/// ==================================================
+///  COMPONENT STYLES
+/// ==================================================
+
+.hxBtn,
+a[href].hxBtn {
+  @include button-reset();
+  @include button-common();
+  @include button-secondary();
+  @include button-medium();
+
+  // ----- SIZE VARIANTS -----
+  &.hxSm {
+    @include button-small();
+  }
+
+  &.hxLg {
+    @include button-large();
+  }
+
+  // ----- STYLE VARIANTS -----
+  &.hxPrimary {
+    @include button-primary();
+  }
+
+  &.hxTertiary {
+    @include button-tertiary();
+  }
+}//.hxBtn
+
+// ========== BUTTON GROUPS ==========
+.hxBtnGroup {
+  border-radius: 2px;
+  display: inline-flex;
+
+  // Allow focus glow to appear
+  &:not(.hxTertiary) {
+    overflow: hidden;
+  }
+
+  > .hxBtn {
+    border-radius: 0;
+
+    + .hxBtn {
+      border-left: 0;
+    }
+  }
+
+  &.hxPrimary {
+    > .hxBtn {
+      @extend .hxBtn.hxPrimary;
```

Comment:
This looks correct, but we'll need to double-check that this is extending in SCSS in the same way it extends in LESS.

The same goes for each `@extend` below, too (compare the CSS in `dist/styles/helix-ui.css` with that of https://unpkg.com/helix-ui@0.18.0-rc.0/dist/styles/helix-ui.css).